tests/test_g2g_communication.py

```diff
@@ -113,9 +113,9 @@ def test_halo_update_only_communicate_on_gpu(backend, gpu_communicators):
     with module_count_calls_to_empty(np), module_count_calls_to_empty(cp):
         shape = (10, 10, 79)
         dims = (fv3gfs.util.X_DIM, fv3gfs.util.Y_DIM, fv3gfs.util.Z_DIM)
-        data = cp.empty(shape, dtype=float)
+        data = cp.ones(shape, dtype=float)
```

Comment:
This was updated to `ones` but the context manager is still `module_count_calls_to_empty`.